src/query/generated/proto/admin/namespace.proto

```diff
@@ -49,3 +49,19 @@ message NamespaceSchemaResetRequest {
 
 message NamespaceSchemaResetResponse {
 }
+
+// NamespaceMarkReadyRequest is the request for transitioning a namespace
+// to the ready state.
+message NamespaceMarkReadyRequest {
+  // Name is the namespace name.
+  string name = 1;
+  // Force forces the StagingStatus of the namespace
+  // to be moved to the ready state.
+  bool force  = 2;
```

Comment:
what happens if you force it if the ns is not actually ready?

Reply:
The coordinator will receive errors. I'll update the comment here to make that more explicit